roles/postgres/files/postgres-backups@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Back up the PostgresSQL database `%i`
+Requires=postgresql.service
+After=postgresql.service
+
+[Service]
+User = postgres
+Group = postgres
+WorkingDirectory=/var/lib/postgresql-backups
+ExecStartPre=+/usr/bin/chown postgres:postgres /var/lib/postgresql-backups
```

Comment:
Q: Why `=+` and not `=`?

Reply:
Oh, it's a a way to run the command before systemd drops the privileges to the ones specified in `User` and `Group` parameters above. We need to run this one as root.